server/lib/nl/detection.py

```diff
@@ -231,6 +238,7 @@ class ClassificationType(IntEnum):
     ClassificationType.CORRELATION,
     ClassificationType.TIME_DELTA,
     ClassificationType.EVENT,
+    ClassificationType.OVERVIEW,
```

Comment:
Can you remove from this?  The ranking here I think could result in not fulfilling the query when we run with this change

Reply:
(might be worth testing what happens with the query)

Reply:
Removed! Tested and "tell me about tulare county" and "tell me about palo alto" work after removal.